### PR TITLE
Fix docker run commands not logging output

### PR DESCRIPTION
**Pull request by robsdedude**
Broke in https://github.com/neo4j-drivers/testkit/pull/527

### CODE REVIEW

File: docker.py [hunk 1]
```diff
@@ -40,6 +45,7 @@ def run(self):
                 err_path = os.path.join(log_path, "err.log")
                 with open(out_path, "a") as out_fd:
                     with open(err_path, "a") as err_fd:
+                        kwargs.update(stdout=out_fd, stderr=err_fd)
```
Comment (thelonelyvulpes) [inline docker.py:48]: why do we change kwargs after it is used?

**Review by thelonelyvulpes — COMMENTED** [review]
makes sense. little unsure about line 48

File: docker.py (hunk 1 above)
Comment (robsdedude) [reply docker.py:48]: Ah :see_no_evil: you spotted how ugly this is. I didn't feel like making it nice. But now I feel exposed and want to.
For explanation `kwargs` is a dict that bound to the whole scope of the `_subprocess_run` function and `_process_run` uses those kwargs. So I'm modifying the `kwargs` dict in place to change what `_process_run` does. It's as ugly and hard to follow as using global variables :nauseated_face:

**Review by thelonelyvulpes — APPROVED** [review]

File: docker.py (hunk 1 above)
Comment (thelonelyvulpes) [reply docker.py:48]: Makes sense, thank you for the explanation.
